memory/libhugetlbfs.py

```diff
@@ -191,7 +191,8 @@ def test(self):
 
     def tearDown(self):
         if self.hugetlbfs_dir:
-            process.system('umount %s' % self.hugetlbfs_dir, sudo=True)
+            process.system('umount %s' % self.hugetlbfs_dir,
+                           sudo=True, ignore_status=True)
```

Comment:
May be a warning would be sufficient here.

Reply:
are you saying we need to have a logger after that if command succeed or failed

Reply:
yes, would be better to let the user know if the command failed.